notifications_api_common/models.py

```diff
@@ -40,6 +40,13 @@ class NotificationsConfig(SingletonModel):
         help_text=_("An upper limit in seconds to the exponential backoff time."),
         default=48,
     )
+    notification_delivery_base_factor = models.PositiveIntegerField(
+        help_text=_(
+            "The base factor used for exponential backoff. Default is 2 (binary), "
+            "but can be increased (e.g., 3 or 4) to spread retries over a longer time period."
+        ),
+        default=2,
```

Comment:
Alex also mentioned to make sure that the defaults for retries make sure 24 hours are covered with 6 to 8 retries https://github.com/open-zaak/open-notificaties/issues/290#issuecomment-2976001824

Could you change the default of this (and the other fields) to make sure this is the case?